src/NServiceBus.Persistence.Sql.Analyzer/SagaMetadataGenerator.cs

```diff
@@ -243,4 +249,6 @@ static void WriteProperty(StringBuilder b, string property, string? value, bool
 
     record SagaDetails(string SagaType, CorrelationDetails? CorrelationProperty, CorrelationDetails? TransitionalProperty, string? TableSuffix);
     readonly record struct CorrelationDetails(string Type, string Name);
+
+    const string DisableGeneratorPropertyName = "build_property.NServiceBusDisableSagaSourceGenerator";
```

Comment:
```suggestion
    const string DisableGeneratorPropertyName = "build_property.NServiceBusSqlPersistenceDisableSagaSourceGenerator";
```

just in case we need to disable source gen in Core saga source generator too?

Which brings up a good point which nomenclature  should we be following?

MSBuild property names are XML element names, so:
	•	They must be valid XML element names
	•	No spaces
	•	No hyphens
	•	No leading digits
	•	Stick to letters, digits, and underscore, starting with a letter or underscore

Sou should we even embark on something like `NServiceBus_Component_Property`?

Reply:
Yeah also not a big fan of underscores either.